pages/student/index.php

```diff
@@ -20,11 +20,25 @@
     </header>
     <?php 
     require_once('../../php/connection.php');
+    require_once(__DIR__ . '/../../php/dbfuncs.php');
```

Comment:
The connection file is required in the dbfuncs file, so you only need the dbfuncs require line.

Reply:
Sorted